.pre-commit-config.yaml

```diff
@@ -1,8 +1,29 @@
+fail_fast: true
+minimum_pre_commit_version: "2"
+
 repos:
   - repo: local
     hooks:
       - id: black
         name: black
         entry: make black
-        language: python
-        always_run: True
+        files: "fidesctl/"
+        language: system
```

Comment:
I'm not seeing `language` as a key option in [the `hooks` documentation](https://pre-commit.com/#pre-commit-configyaml---hooks)...?

Reply:
[this section here](https://pre-commit.com/#creating-new-hooks) talks about it, the other section is for pre-defined hooks